src/main/resources/static/common/js/demoAdmin.js

```diff
@@ -109,6 +109,55 @@ mod.controller('demo_admin_controller', function($rootScope, $scope, $http, $fil
         }
     }
 
+    demo.loadGRPCConfigurationDetails = function() {
+        $http({
+            method: 'GET',
+            url: '/v1/demoAdmin/gRPCProperties',
+        }).then(function(result) {
+            const data = result.data.data
+            demo.gRPCConfigurationJsonDetails = [
```

Comment:
There are some UI lags when opening the modal dialog. I think the only information we need from the callback is the port. So please set other values before we make the API call and only set port value in the callback to improve the performance.